components/dashboard/src/components/ContextMenu.tsx

```diff
@@ -84,7 +84,7 @@ function ContextMenu(props: ContextMenuProps) {
                 <div className={`mt-2 z-50 bg-white dark:bg-gray-900 absolute flex flex-col border border-gray-200 dark:border-gray-800 rounded-lg truncated ${props.classes || 'w-48 right-0'}`}>
                     {props.menuEntries.map((e, index) => {
                         const clickable = e.href || e.onClick || e.link;
-                        const entry = <div className={`px-4 flex py-3 ${clickable ? 'hover:bg-gray-200 dark:hover:bg-gray-800' : ''} text-sm leading-1 ${e.customFontStyle || font} ${e.separator ? ' border-b border-gray-200 dark:border-gray-800' : ''}`} title={e.title}>
+                        const entry = <div className={`px-4 flex py-3 ${clickable ? 'hover:bg-gray-100 dark:hover:bg-gray-800' : ''} ${e.active ? 'bg-gray-50 dark:bg-gray-800' : ''} ${index === 0 ? 'rounded-t-lg' : ''} ${index === props.menuEntries.length - 1 ? 'rounded-b-lg' : ''} text-sm leading-1 ${e.customFontStyle || font} ${e.separator ? ' border-b border-gray-200 dark:border-gray-800' : ''}`} title={e.title}>
```

Comment:
**issue**: This is minor but there's a small gap on the corners due to corner radiuses. It should be barely noticable but if there's an easy fix let's also include it here. Otherwise, a follow up issue could suffice. 🏓

Reply:
Oh no 🙈 that's so unsatisfying 😬

Reply:
Follow-up issue: https://github.com/gitpod-io/gitpod/issues/5086